src/main/resources/hudson/tasks/Mailer/global.properties

```diff
@@ -0,0 +1,22 @@
+# The MIT License
```

Comment:
@ndeloof Why is this empty "global.properties" file added for `Mailer` but no "config.properties" for `SMTPAuthentication`?